packages/next-swc/crates/next-core/src/manifest.rs

```diff
@@ -43,16 +43,16 @@ impl DevManifestContentSourceVc {
 
         async fn content_source_to_pathname(
             content_source: ContentSourceVc,
-        ) -> Result<Option<String>> {
+        ) -> Result<Option<StringReadRef>> {
             // TODO This shouldn't use casts but an public api instead
             if let Some(api_source) = NodeApiContentSourceVc::resolve_from(content_source).await? {
-                return Ok(Some(format!("/{}", api_source.get_pathname().await?)));
+                return Ok(Some(api_source.get_pathname().await?));
```

Comment:
Don't we still need the / prefix in the manifest?

Reply:
no pathname already includes a /